firedrake/functionspaceimpl.py

```diff
@@ -391,15 +391,15 @@ def __mul__(self, other):
 
     @property
     def node_count(self):
-        """The number of global nodes in the function space.  If the
+        """The number of local nodes in the function space.  If the
```

Comment:
I think there might be a degree of confusion about nomenclature here. I think "local" means "per element".